src/js/apps/forms/form/form_app.js

```diff
@@ -141,75 +137,46 @@ export default App.extend({
     response.set({ _created_at: dayjs().format() });
     this.responses.unshift(response);
     this.setState({ responseId: response.id });
+  },
+  onFormServiceError() {
+    if (this.loadingModal) this.loadingModal.destroy();
 
-    this.showFormStatus();
-    this.showFormUpdate();
+    this.showFormSave();
   },
   onFormServiceReady() {
     this.showFormSave();
   },
   onFormServiceUpdateSubmission(updated) {
     this.showLastUpdated(updated);
   },
-  onFormServiceError() {
-    if (this.loadingModal) this.loadingModal.destroy();
-
-    this.showFormSave();
-  },
   stateEvents: {
     'change': 'onChangeState',
-    'change:shouldShowHistory': 'showFormAction',
+    'change:isExpanded': 'showSidebar',
+    'change:isActionSidebar': 'showSidebar',
+    'change:shouldShowHistory': 'showFormActions',
     'change:responseId': 'onChangeResponseId',
-    'change:saveButtonType': 'onChangeSaveButtonType',
   },
   onChangeState(state) {
-    if (!state.hasChanged('isExpanded') && !state.hasChanged('isActionSidebar')) return;
-
-    this.showSidebar();
-
-    store.set(`form-state_${ this.currentUser.id }`, {
-      isExpanded: this.getState('isExpanded'),
-      saveButtonType: this.getState('saveButtonType'),
-    });
+    store.set(`form-state_${ this.currentUser.id }`, state.pick('isExpanded', 'saveButtonType'));
   },
   onChangeResponseId() {
+    this.showFormActions();
     this.showContent();
-    this.showFormStatus();
-  },
-  onChangeSaveButtonType() {
-    store.set(`form-state_${ this.currentUser.id }`, {
-      isExpanded: this.getState('isExpanded'),
-      saveButtonType: this.getState('saveButtonType'),
-    });
-  },
-  showFormStatus() {
-    if (this.isReadOnly) return;
-    this.showChildView('status', new StatusView({
-      model: this.responses.first(),
-      isEditing: !this.getState('responseId'),
-    }));
-  },
-  showLastUpdated(updated) {
-    if (this.isReadOnly || this.getState('responseId')) return;
-
-    const lastUpdatedView = new LastUpdatedView({ updated });
-
-    this.showChildView('formUpdated', lastUpdatedView);
   },
-  showActions() {
-    const formActions = new FormActionsView({
+  showStateActions() {
```

Comment:
I renamed this to state actions to differentiate from the other actions as these things modify stuff on the state model